### ISSUE TITLE
Support JSON In MultiType

### ISSUE THREAD

**Issue by effs** [E1337Kat/cyberpunk2077_ext_redux#135]
What it says on the tin. Just creating a separate issue since it's better tracked away from the XML work I started it in. But is basically required for that to be considered complete.

**Comment by effs** [comment]
Toplevel subdir CET fails on JSON, should be able to fix that here https://www.nexusmods.com/cyberpunk2077/mods/3301?tab=files

**Comment by effs** [comment]
📁 ChangeCTPPCamera
📄 config.lua (1.4 kB)
📄 init.lua (49.4 kB)
📄 tppvalues.json (134 B)

**Comment by effs** [comment]
This will be installable after #137
